.travis.yml

```diff
@@ -4,8 +4,15 @@ jdk:
   - openjdk8
   - openjdk11
 addons:
-  postgresql: "9.4"
-  firefox: "60.3.0esr"
+  postgresql: "11"
+  apt:
+    packages:
+      - postgresql-11
+      - postgresql-client-11
+  firefox: "68.2.0esr"
```

Comment:
I've been using a fixed version of firefox because I'm using selenium 2.x and the latest version causes a problem.
Now that we are already on selenium 3.x, we don't need to set the firefox version, so we will change it to `latest-esr`.
c8ab13a71063f1b2cb1f1e5381db2c206261b148